### PR TITLE
Renaming workbook upload

### PR DESCRIPTION
**Pull request by mhpjay422**
Why this PR
- To rename all instances of workbook upload to project onboarding

This PR contains: 
- Renaming of links, logic and UI to replace workbook upload with project onboarding
- Added migration to change default value

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -12,7 +12,7 @@ ONBOARDING_CLIENT_ID=...
 ONBOARDING_CLIENT_SECRET=...
 ONBOARDING_ENVIRONMENT_ID=us_env_...
 ONBOARDING_SPACE_CONFIG_ID=us_sc_...
-WORKBOOK_UPLOAD_WORKBOOK_NAME="HCM Workbook"
+PROJECT_ONBOARDING_WORKBOOK_NAME=MyWorkbook123
```
Comment (johnmosesman) [inline .env.example:15]: If we're changing this just `ONBOARDING_` matches the prefixes above

File: pages/project-onboarding.tsx [hunk 2]
```diff
@@ -311,7 +311,7 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
     where: {
       userId_type: {
         userId: token.sub as string,
-        type: SpaceType.WorkbookUpload,
+        type: SpaceType.ProjectOnboarding,
```
Comment (johnmosesman) [inline pages/project-onboarding.tsx:314]: What happens here after we roll this out with the existing database values that are the old values?

File: prisma/migrations/20230413140043_update_space_type_default/migration.sql [hunk 3]
```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE "Space" ALTER COLUMN "type" SET DEFAULT 'project onboarding';
```
[the remark this answers is not in the document]
Comment (mhpjay422) [reply prisma/migrations/20230413140043_update_space_type_default/migration.sql:2]: @johnmosesman - The previous migration had a default value set. I'll change to no default?
CleanShot 2023-04-13 at 16 02 36

[the remark this answers is not in the document]
Comment (mhpjay422) [reply prisma/migrations/20230413140043_update_space_type_default/migration.sql:2]: Deleted comment. Wasnt reading it right

**Comment by mhpjay422** [comment]
CleanShot 2023-04-13 at 16 23 17
CleanShot 2023-04-13 at 16 25 29

File: .env.example [hunk 4]
```diff
@@ -12,7 +12,7 @@ ONBOARDING_CLIENT_ID=...
 ONBOARDING_CLIENT_SECRET=...
 ONBOARDING_ENVIRONMENT_ID=us_env_...
 ONBOARDING_SPACE_CONFIG_ID=us_sc_...
-WORKBOOK_UPLOAD_WORKBOOK_NAME="HCM Workbook"
+ONBOARDING_WORKBOOK_NAME=MyWorkbook123
```
Comment (johnmosesman) [inline .env.example:15]: @mhpjay422 Before merging add this new var to heroku, and after it's deployed remove the old one

**Review by johnmosesman — APPROVED** [review]
see note on env vars

File: .env.example (hunk 4 above)
Comment (mhpjay422) [reply .env.example:15]: added new var to heroku

Comment (mhpjay422) [reply .env.example:15]: deployed and removed the old one

**Pull request merged**